WordPress/src/main/res/layout/note_block_user.xml

```diff
@@ -1,84 +1,78 @@
 <?xml version="1.0" encoding="utf-8"?>
+
+
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/user_block_root_view"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@android:color/white">
+    android:background="?android:selectableItemBackground"
+    android:clickable="true"
+    android:contentDescription="@null"
+    android:focusable="true"
+    android:orientation="vertical"
+    android:paddingStart="@dimen/margin_extra_large"
+    android:paddingEnd="@dimen/margin_extra_large">
 
     <LinearLayout
-        android:contentDescription="@null"
-        android:focusable="true"
-        android:clickable="true"
-        android:id="@+id/user_block_root_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:background="?android:selectableItemBackground"
-        android:paddingStart="@dimen/margin_extra_large"
-        android:paddingEnd="@dimen/margin_extra_large">
+        android:orientation="horizontal"
+        android:paddingTop="@dimen/margin_medium"
+        android:paddingBottom="@dimen/margin_medium">
+
+        <ImageView
+            android:id="@+id/user_avatar"
+            android:layout_width="@dimen/notifications_avatar_sz"
+            android:layout_height="@dimen/notifications_avatar_sz"
+            android:layout_gravity="center_vertical"
+            android:layout_marginEnd="@dimen/notifications_adjusted_font_margin"
+            android:importantForAccessibility="no" />
 
         <LinearLayout
-            android:layout_width="match_parent"
+            android:layout_width="fill_parent"
```

Comment:
Same thing as before: shouldn't we use `match_parent` instead of `fill_parent`?

```suggestion
            android:layout_width="match_parent"
```

Reply:
`fill_parent` is a blast from the past for sure :)